src/electionguard_cli/e2e_steps/input_retrieval_step.py

```diff
@@ -0,0 +1,66 @@
+from typing import List, Optional
+from cli_models.e2e_inputs import E2eInputs
+from e2e_steps.e2e_step_base import E2eStepBase
+
+from electionguard.ballot import PlaintextBallot
+from electionguard.guardian import Guardian
+from electionguard.manifest import InternationalizedText, Manifest
+from electionguard_tools.factories.ballot_factory import BallotFactory
+from electionguard_tools.factories.election_factory import (
+    ElectionFactory,
+)
+
+
+class InputRetrievalStep(E2eStepBase):
+    """Responsible for retrieving and displaying user provided inputs"""
```

Comment:
This is a nitpick, but more for informing. I believe the standard docstyle uses `.` at the end of these lines. I turned off the line formatting for this because it was annoying with some of existing files, but figured I'd catch if for the future.